src/views/Player.vue

```diff
@@ -153,11 +164,44 @@ export default {
       return dateTime
     },
 
+    waitFor(conditionFunction) {
+      const poll = resolve => {
+        if(conditionFunction()) resolve();
+        else setTimeout(() => poll(resolve), 400);
+      }
+      return new Promise(poll);
+    },
+
     logData() {
       if (this.plioId != undefined && this.player.playing) this.uploadJson()
       TIMEOUT = setTimeout(this.logData, UPLOAD_INTERVAL)
     },
 
+    startTheVideo() {
+      var x = document.getElementById('start-button')
+      x.classList.remove('start-button')
+      x.classList.add('start-button-active')
+      setTimeout(() => {
+        this.player.fullscreen.enter();
+        this.waitFor(() => this.player.fullscreen.active === true)
+        .then(() => this.player.play())
+      }, 400);
+
+      this.tutorialProgress['start'] = true;
+      this.checkIfTutorialIsComplete();
+    },
+
+    checkIfTutorialIsComplete(){
+      this.isTutorialComplete = true;
+      for (const [key, value] of Object.entries(this.tutorialProgress)) {
+        console.log(key, value);
+        if (value == false) {
+          this.isTutorialComplete = false;
+          return
+        }
+      }
+    },
```

Comment:
iterate through the dictionary,
if all values are true, then the tutorial is complete and vice versa